### PR TITLE
Datetime Column [issue-22]

### PR DESCRIPTION
**Pull request by JaySunSyn**
Please merge https://github.com/Neovici/cosmoz-omnitable/pull/38 first.

### CODE REVIEW

File: cosmoz-omnitable-column-datetime.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+			<paper-input no-label-float type="datetime-local" on-change="_dateValueChanged"
+				value="[[ renderDateValue(item, valuePath, userFormat, locale) ]]">
+			</paper-input>
+		</template>
+
+		<template class="header">
+			<paper-dropdown-menu label="[[ title ]]" placeholder="[[ _filterText ]]" title="[[ _tooltip ]]">
+				<div class="dropdown-content" slot="dropdown-content" style="padding: 15px; min-width: 100px;">
+					<h3 style="margin: 0;">[[ title ]]</h3>
+					<paper-input type="datetime-local" min="[[ _formatISODate(_possibleDateRange.min) ]]" max="[[ _formatISODate(_latestAfter) ]]"
```
Comment (nomego) [inline cosmoz-omnitable-column-datetime.html]: Not supported in Safari (or Firefox) ?

**Comment by JaySunSyn** [comment]
Can be merged.

**Comment by nomego** [comment]
Sorry I didn't have a look at this, it will need to be rebased and the min/max support carried over to the behavior.
Also, what is the plan for datetime-local, proof-of-concept or something you intend to maintain on your own?

**Comment by JaySunSyn** [comment]
Regarding `datetime-local` I actually asked for access to a new repo :)
Or would you integrate it into `cosmoz-omnitable`?

**Pull request merged**